File: nathan/Problema0168/problema0168.py
contPrimos = 0

for x in range(10):
    n = float(input("Digite o um número: "))
    if n % 2 == 0 or n % 3 == 0:
        if n == 2 or n == 3:
            contPrimos = contPrimos + 1
    else:
        contPrimos = contPrimos + 1
print("Quantidade de números primos: ", contPrimos)

File: nathan/Problema0168/problema0168.java
package Problema0168;

import java.util.Scanner;

public class problema0168{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
     
        int contPrimo = 0;

        for(int i = 0; i < 10; i++){
            System.out.println("Digite o " + (i+1) +" número: ");
            double n = Double.valueOf(input.nextLine());
            if(n % 2 == 0 || n % 3 == 0){
                if(n == 2 || n == 3)
                    contPrimo++;
            }else
                contPrimo++;
        }
        System.out.println("Quantidade de primos: " + contPrimo);
    }
}
